pages/en/developing/supported-networks.mdx

```diff
@@ -66,6 +66,8 @@ Ropsten, Rinkeby and Kovan are being deprecated. Read more on the [Ethereum Foun
 
 Subgraphs indexing Gnosis Chain can now be deployed with the `gnosis` network identifier. `xdai` is still supported for existing hosted service subgraphs.
 
+For a full list of which features are supported on different networks, see [this page](https://github.com/graphprotocol/indexer/blob/main/docs/networks/mainnet.md#feature-support).
```

Comment:
The features include, but are not limited to, network support. It also includes features such as grafting, non-fatal errors and fulltext search. So I would prefer that we change this sentence to 

For a full list of which features are supported on the decentralized network, see [this page] (https://github.com/graphprotocol/indexer/blob/main/docs/networks/mainnet.md#feature-support).